pkg/clustermesh/remote_cluster_test.go

```diff
@@ -234,3 +237,132 @@ func TestIPCacheWatcherOpts(t *testing.T) {
 		})
 	}
 }
+
+func TestRemoteClusterValidate(t *testing.T) {
```

Comment:
This test could be likely moved alongside the `ClusterInfo.ValidateRemoteConfig` definition.